### DISCUSSION TITLE
Trouble installing whisper

### DISCUSSION THREAD

**Discussion by Bakarel** [openai/whisper#78] [Q&A] [answered]
I was trying to install it on windows though Msys2 using this pip package (https://packages.msys2.org/base/python-pip).

My log of what happened:

>  pip install git+https://github.com/openai/whisper.git
> Collecting git+https://github.com/openai/whisper.git
>   Cloning https://github.com/openai/whisper.git to /tmp/pip-req-build-53u_63i_
>   Running command git clone --filter=blob:none --quiet https://github.com/openai/whisper.git /tmp/pip-req-build-53u_63i_
>   Resolved https://github.com/openai/whisper.git to commit 15ab54826343c27cfaf44ce31e9c8fb63d0aa775
>   Preparing metadata (setup.py) ... done
> Collecting numpy
>   Using cached numpy-1.23.3.tar.gz (10.7 MB)
>   Installing build dependencies ... done
>   Getting requirements to build wheel ... done
>   Preparing metadata (pyproject.toml) ... done
> ERROR: Could not find a version that satisfies the requirement torch (from whisper) (from versions: none)
> ERROR: No matching distribution found for torch

If someone can explain what I did wrong that would be appreciated.

**Comment by jongwook** [comment, marked as answer]
It seems like PyTorch does not provide pre-built packages for msys2. Can you try [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) instead?

**Reply by tantan39** [reply]
WSL is only on Windows, how about macOS?

**Comment by Bakarel** [comment]
Thanks very much that works.
